tests/README.md

```diff
@@ -0,0 +1,31 @@
+## How run integration tests
+
+### Run tests locally
+
+Copy example files and update corresponding values.
+
+```shell
+cp Makefile.env.exmaple Makefile.env
+```
+
+Run tests
+
+```shell
+make integration_test
+```
+
+### Run tests in CI
+
+Set following environment variables:
+
+```shell
+export STORAGE_IPFS_INTEGRATION_TEST=on
+export STORAGE_IPFS_NAME=ipfsnodename
```

Comment:
ditto

Reply:
And actually, there's a series of minor problems in setting up integration tests. Here's the tracking issue: https://github.com/beyondstorage/go-service-example/issues/6

You can come and express your opinion if you like.